docs-src/python/how-to-asynchronously-complete-an-activity-in-python.md

```diff
@@ -0,0 +1,29 @@
+---
+id: how-to-asynchronously-complete-an-activity-in-python
+title: How to asynchronously complete an Activity in Python
+sidebar_label: Asynchronous Activity Completion
+description: To enable an Activity Function to return without the Activity Execution completing, call the `cancel` task object.
+tags:
+  - how-to
+  - python
+---
+
+To cancel an asynchronous Activity, call the [`cancel`](https://docs.python.org/3/library/asyncio-task.html#asyncio.Task.cancel) task object.
```

Comment:
Asynchronous activity completion is unrelated to cancellation. I think this code and documentation isn't what should be present for "Asynchronous Activity Completion". Asynchronous activity completion is about returning from an activity without completing it and then completing it later (I put code in #1454 relating to each of those two parts).

Reply:
Yes, this is exactly what I called out before as well.